tests/cli/functional/actions/test_auth_actions.py

```diff
@@ -39,79 +39,67 @@
 
 
 @pytest.mark.parametrize('confirm', (True, False))
-def test_get_password_from_prompt_cli_action(mock_click_prompt, mock_click_confirm, confirm):
+def test_get_password_from_prompt_cli_action(mocker, mock_stdin, confirm, capsys):
 
     # Setup
-    mock_click_prompt.return_value = INSECURE_DEVELOPMENT_PASSWORD
-    mock_click_confirm.return_value = True
-
-    # Test
-    result = get_password_from_prompt(confirm=confirm)
-    assert result
-    assert result is not NO_PASSWORD
-    assert result == INSECURE_DEVELOPMENT_PASSWORD
-
-    mock_click_prompt.assert_called_once_with(GENERIC_PASSWORD_PROMPT,
-                                              confirmation_prompt=confirm,
-                                              hide_input=True)
-
-
-@pytest.mark.parametrize('confirm', (True, False))
-def test_get_password_from_prompt_cli_action(mocker, mock_click_prompt, mock_click_confirm, confirm):
-
-    # Setup
-    mock_click_prompt.return_value = INSECURE_DEVELOPMENT_PASSWORD
-    mock_click_confirm.return_value = True
-    test_envavr = 'NUCYPHER_TEST_ENVVAR'
+    mock_stdin.password(INSECURE_DEVELOPMENT_PASSWORD, confirm=confirm)
+    test_envvar = 'NUCYPHER_TEST_ENVVAR'
     another_password = 'th1s-iS-n0t-secur3'
 
-    mocker.patch.dict(os.environ, {test_envavr: another_password})
+    mocker.patch.dict(os.environ, {test_envvar: another_password})
     result = get_password_from_prompt(confirm=confirm)
     assert result == INSECURE_DEVELOPMENT_PASSWORD
-    mock_click_prompt.assert_called_once_with(GENERIC_PASSWORD_PROMPT,
-                                              confirmation_prompt=confirm,
-                                              hide_input=True)
-
-    mock_click_prompt.reset()
+    assert mock_stdin.empty()
+    captured = capsys.readouterr()
+    assert GENERIC_PASSWORD_PROMPT in captured.out
+    if confirm:
+        assert "Repeat for confirmation:" in captured.out
 
     # From env var
-    mocker.patch.dict(os.environ, {test_envavr: another_password})
-    result = get_password_from_prompt(confirm=confirm, envvar=test_envavr)
+    mocker.patch.dict(os.environ, {test_envvar: another_password})
+    result = get_password_from_prompt(confirm=confirm, envvar=test_envvar)
     assert result is not NO_PASSWORD
     assert result != INSECURE_DEVELOPMENT_PASSWORD
     assert result == another_password
-    mock_click_prompt.assert_called_once_with(GENERIC_PASSWORD_PROMPT,
-                                              confirmation_prompt=confirm,
-                                              hide_input=True)
+    assert mock_stdin.empty()
+    captured = capsys.readouterr()
+    assert not captured.out
```

Comment:
Do we want to check stderr here as well?
```suggestion
    assert not captured.out
    assert not captured.err
```

Reply:
Fixed.